storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -286,34 +288,32 @@ public void doRegistration(final SchedulerDriver driver, Protos.FrameworkID id)
     _state.put(FRAMEWORK_ID, id.getValue());
     _offers = new HashMap<Protos.OfferID, Protos.Offer>();
 
-    if (_enabledLogviewerSidecar) {
-
-      _timer.scheduleAtFixedRate(new TimerTask() {
-        @Override
-        public void run() {
-          // performing "explicit" reconciliation; master will respond with the latest state for all logviewer tasks
-          // in the framework scheduler's statusUpdate() method
-          List<TaskStatus> taskStatuses = new ArrayList<TaskStatus>();
-          List<String> logviewerPaths = _zkClient.getChildren(_logviewerZkDir);
-          if (logviewerPaths == null) {
-            _driver.reconcileTasks(taskStatuses);
-            return;
-          }
-          for (String path : logviewerPaths) {
-            TaskID logviewerTaskId = TaskID.newBuilder()
-                                           .setValue(new String(_zkClient.getNodeData(String.format("%s/%s", _logviewerZkDir, path))))
-                                           .build();
-            TaskStatus logviewerTaskStatus = TaskStatus.newBuilder()
-                                                       .setTaskId(logviewerTaskId)
-                                                       .setState(TaskState.TASK_RUNNING)
-                                                       .build();
-            taskStatuses.add(logviewerTaskStatus);
-          }
+    _timer.scheduleAtFixedRate(new TimerTask() {
```

Comment:
align this with the rest of the code please

Reply:
This is actually aligned with the rest of the code. The diff makes it seem like it isn't. I'm taking the timer thread out of the `if (_enabledLogviewerSidecar)` block and scheduling it unconditionally. See https://github.com/srikanth-viswanathan/mesos-storm/blob/66dcbbe502aa3438b2a1b107d146e25de058f82c/storm/src/main/storm/mesos/MesosNimbus.java#L291 for the full file.